CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,134 @@
+
```

Comment:
Can you remove this first blank line please?

I know it's in the example but I don't think it's necessary :sweat_smile: